src/core/withExperiment.js

```diff
@@ -108,7 +108,7 @@ export const withExperiment = ({
       const isExperimentEnabled = this.isExperimentEnabled();
 
       const exposedProps: WithExperimentInjectedProps = {
-        experimentEnabled: isExperimentEnabled,
+        experimentIsEnabled: isExperimentEnabled,
```

Comment:
heh, now I think we could use the exact same name everywhere, so maybe pick on of the two here and adjust the rest of the code (either change the HOC `isExperimentEnabled` var/method name or the other way around)

Reply:
Good point. I've changed everything everywhere to just use `isExperimentEnabled`.